SocialFilter/sections/css_extended.txt

```diff
@@ -7,6 +7,7 @@
 !
 academics.de##strong:contains(/^Diesen Job teilen$/)
 mobile-review.com#?#p:has(> strong:contains(Поделиться))
+yodobashi.com##.pDetailCaption:has( > div.pDetailSNS)
```

Comment:
You should add this `yodobashi.com#?#.pDetailCaption:has(> div.pDetailSNS)`.
Your rule has some mistakes.

Reply:
sorry, and thank you for fixed my bad rule.

Reply:
You are welcome :)